lib/logger/lib/logger.ex

```diff
@@ -939,8 +939,14 @@ defmodule Logger do
   def __should_log__(level, module) do
     level = elixir_level_to_erlang_level(level)
 
-    if :logger_config.allow(level, module) do
-      level
+    case get_process_level(self()) do
```

Comment:
We need to apply the same to  `bare_log` after the recent commits and add tests :)